charts/kafka-operator/values.yaml

```diff
@@ -48,13 +51,20 @@ prometheusMetrics:
       repository: gcr.io/kubebuilder/kube-rbac-proxy
       tag: v0.4.0
       pullPolicy: IfNotPresent
+    serviceAccount:
+      create: true
+      name: kafka-operator-authproxy
+
 
 nameOverride: ""
 fullnameOverride: ""
 
 rbac:
   enabled: true
 
+crd:
+  enabled: false
```

Comment:
let's make sure this is not breaking with previous version. should this be `true` ?

Reply:
fair, yeah, by default should be true, fixing